api/net/Areas/Subscriber/Controllers/WorkOrderController.cs

```diff
@@ -67,14 +67,16 @@ public class WorkOrderController : ControllerBase
     /// <returns></returns>
     [HttpGet]
     [Produces(MediaTypeNames.Application.Json)]
-    [ProducesResponseType(typeof(IPaged<WorkOrderMessageModel>), (int)HttpStatusCode.OK)]
+    [ProducesResponseType(typeof(IPaged<WorkOrderModel>), (int)HttpStatusCode.OK)]
```

Comment:
I don't know why I was using the WorkOrderMessageModel.  I hope this doesn't introduce a regression issue.